### PR TITLE
604-separate project scss

### PR DESCRIPTION
**Pull request by fdiblen**
# Related
Refs: #604

# Description
This PR separates project and software page styles. The element styles in frontend/style/templates/_software.scss moved to frontend/style/templates/_project.scss

# Screenshots:

<details>
<summary>Example project page view using master branch</summary>

localhost_projects_764__master

</details>

<details>
<summary>Example project page with the new scss</summary>

localhost_projects_764__new_css

</details>

# Testing:
Follow the steps below to build and start the stack. 
```
git checkout 604-project-scss-new
docker-compose down --volumes --remove-orphan
docker-compose build frontend
docker-compose up -d
docker-compose logs --follow
```

Visit [https://localhost/projects/764](https://localhost/projects/764) or another project page to see the result.

### CODE REVIEW

**Review by sverhoeven — COMMENTED** [review]
Your pdfs contain weird scroll artifacts. 

You can use 
1. DevTools (F12) 
1. CTRL-SHIFT-p 
1. Search for `screenshot` to make a full size screenshot

**Review by sverhoeven — CHANGES_REQUESTED** [review]
I did some blink testing, by switching rapidly between old and new page in web browser tabs. First in desktop screen size and then in ipad and pixel xl screen size.

## /

Looks same

## /projects

Looks same

## /software/xenon

<details>
<summary>Broken</summary>

localhost_software_xenon__css-perproject-604

</details>
Probably due to commented out line in rsd.scss

## /projects/764

<details>
<summary>Divider below related tools is missing</summary>

localhost_projects_764_css-perproject-604

</details>

* Output / Impact / Related projects headings are left aligned on narrow screen instead of centered, but that's OK
* The white space between related projects and related tools is smaller, but that's OK.

Other suggestions
* Can you merge #629 PR into this branch, as there is a regression with the project image on narrow screens.

**Comment by sverhoeven** [comment]
I compared this PR against commit https://github.com/research-software-directory/research-software-directory/commit/450563a3ed378a8d55f55d15612b4fc629054c9e on master branch

**Comment by fdiblen** [comment]
> Your pdfs contain weird scroll artifacts.
> 
> You can use
> 
> 1. DevTools (F12)
> 2. CTRL-SHIFT-p
> 3. Search for `screenshot` to make a full size screenshot

Just did print the page :) Your suggestion is better.

**Comment by fdiblen** [comment]
> I did some blink testing, by switching rapidly between old and new page in web browser tabs. First in desktop screen size and then in ipad and pixel xl screen size.
> 
> ## /
> Looks same
> 
> ## /projects
> Looks same
> 
> ## /software/xenon
> Broken
> Probably due to commented out line in rsd.scss
> ## /projects/764
> Divider below related tools is missing
> * Output / Impact / Related projects headings are left aligned on narrow screen instead of centered, but that's OK
> * The white space between related projects and related tools is smaller, but that's OK.
> 
> Other suggestions
> 
> * Can you merge #629 PR into this branch, as there is a regression with the project image on narrow screens.

Thank you for reviewing @sverhoeven!
All the issues you indicated are fixed. Can you please have a look at it again?

File: frontend/templates/project/project_index.html [hunk 1]
```diff
@@ -4,38 +4,38 @@
...
-				        <p class="text-blue">{{ title.title }} ({{ title.yearStart }}{% if title.yearStart != title.yearEnd %}-{{ title.yearEnd }}{% endif %})</p>
-				        <p>{{title.subtitle}}</p>
-				    </div>
-			    </div>
-		        </a></article>
-                    {% endfor %}
+<div id="project_page">
+	<section class="" id="title">
+		<div class="container">
+			<div class="row">
```
Comment (sverhoeven) [inline frontend/templates/project/project_index.html:10]: Can you correct the indentation, it looks a bit sloppy now

**Review by sverhoeven — APPROVED** [review]
# /

Still looks same

# /software/xenon

* Organization logos are bigger, but that's OK

# /projects

Still looks the same.

# /projects/76

Now looks the same

Overall good job, just some minor indentation problems.

File: frontend/templates/project/project_index.html (hunk 1 above)
Comment (fdiblen) [reply frontend/templates/project/project_index.html:10]: done

Comment (sverhoeven) [reply frontend/templates/project/project_index.html:10]: thanks

**Pull request merged**